pkg/service/healthserver/healthserver.go

```diff
@@ -189,6 +190,13 @@ func (h *httpHealthHTTPServerFactory) newHTTPHealthServer(port uint16, svc *Serv
 
 		if err := srv.ListenAndServe(); err != http.ErrServerClosed {
 			svc := srv.loadService()
+			if strings.Contains(err.Error(), "bind: address already in use") {
+				log.WithError(err).WithFields(logrus.Fields{
+					logfields.ServiceName:                svc.Service.Name,
+					logfields.ServiceNamespace:           svc.Service.Namespace,
+					logfields.ServiceHealthCheckNodePort: port,
+				}).Error("ListenAndServe failed for service health server, since the user might be running with kube-proxy. Please ensure that 'enable-health-check-nodeport' option is set to false if 'kube-proxy-replacement' is set to 'partial' mode.")
```

Comment:
Nit: `Errorf("..... Please ensure that '--%s' option is set to false if '--%s' is set to '%s', option.EnableHealthCheckNodePort, option.KubeProxyReplacement, option.KubeProxyReplacementPartial)`.